homeassistant/components/color_extractor/services.yaml

```diff
@@ -0,0 +1,19 @@
+predominant_color_url:
```

Comment:
Call it `turn_on` so it's clear it's a wrapper around the light.turn_on service

Reply:
appended `_turn_on` to both services

Reply:
Retract my previous, now have one service `color_extract.turn_on` which handles either path